pkg/network/ebpf/c/tracer.h

```diff
@@ -119,6 +119,8 @@ typedef struct {
     // this is useful for detecting race conditions that result in a batch being overrriden
     // before it gets consumed from userspace
     __u64 idx;
+    // current batch slot to be writen to
+    __u8 pos;
```

Comment:
Happy to move away from this `pos` terminology in favor of `len` which is less confusing in a future PR